examples/webpack-modern/webpack.config.babel.js

```diff
@@ -10,6 +10,11 @@ module.exports = {
     path: path.join(__dirname, 'dist'),
     filename: 'bundle.js',
   },
+  resolve: {
+    alias: {
+        'react-dom': path.resolve( path.join( __dirname, '.', 'node_modules', '@hot-loader/react-dom' ) )
```

Comment:
`'react-dom': '@hot-loader/react-dom'` should be enough here if the readme is right